src/utils/recap-onboot.service.in

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=recap on boot
+Documentation=man:recap(5) man:recap(8)
+After=network-online.target
+Wants=network-online.target
```

Comment:
Just asking the question - why network-online.target? Is this to ensure the iproute2 utilities will function correctly, or ensure it's in multi-user mode? Perhaps [basic.target](https://www.freedesktop.org/software/systemd/man/systemd.special.html) makes more "generic" sense if we don't absolutely require networking?

Reply:
I was thinking more of services like mysql or a webserver running, those services require network so is more likely that recap will obtain logs from those services(when needed) if it runs once the netework is "online" per the target.